firebase-app-distribution/api.txt

```diff
@@ -1,15 +0,0 @@
-// Signature format: 2.0
```

Comment:
We should probably regenerate these files instead of deleting the whole thing. It's okay if we end up closing my PR to update the api.txt files because this PR makes those changes obsolete

Reply:
Sounds good.